cmd/skipper/main.go

```diff
@@ -273,6 +275,7 @@ func init() {
 	flag.IntVar(&defaultHTTPStatus, "default-http-status", http.StatusNotFound, defaultHTTPStatusUsage)
 	flag.BoolVar(&suppressRouteUpdateLogs, "suppress-route-update-logs", false, suppressRouteUpdateLogsUsage)
 	flag.BoolVar(&enablePrometheusMetrics, "enable-prometheus-metrics", false, enablePrometheusMetricsUsage)
+	flag.BoolVar(&enableCodahaleMetrics, "enable-codahale-metrics", false, enableCodahaleMetricsUsage)
```

Comment:
Isn't this a breaking change?

Reply:
No it isn't, because the flag is miss leading. ;)
This flag is only required for enabling codahale metrics, if prometheus metrics are enabled.

Reply:
The truth is that if we enable this by default, then if we enable enablePrometheusMetrics, how to decide if enableCodahaleMetrics should be enabled, too?

Maybe it's better to have --metrics-flavour=<codahale|prometheus|all> opinions?

Reply:
How about instead?
--metrics-flavour=codahale, prometheus

Reply:
+1 on @ruiaraujo suggestion, maybe by validating that the strings for the metrics type are actually supported.

Reply:
I agree, I will change that

Reply:
(sry, meant the plus one :D )